src/View/Helper/FormHelper.php

```diff
@@ -543,7 +543,6 @@ protected function _csrfField(): string
         return $this->hidden('_csrfToken', [
             'value' => $csrfToken,
             'secure' => static::SECURE_SKIP,
-            'autocomplete' => 'off',
```

Comment:
This had a specific purpose: https://github.com/cakephp/cakephp/pull/10487 Has this been fixed in Safari by now?

Reply:
I don't see that, I edited it based on the W3C validator.

Reply:
We shouldn't remove this as it could cause problems with safari and other webkit browsers.

Reply:
It's a problem that's over 4 years old, it couldn't have been solved already, don't you know?

Reply:
It could, but we shouldn't make assumptions about it, but have it actually tested. Unless someone can confirm that it's fixed (and has been for quite some time), we shouldn't change this and potentially break peoples stuff when there is no actual technical need to do so, even if custom one-time token implementations might be rather rare.